keyboards/smithrune/iron165r2_h/info.json

```
@@ -0,0 +1,123 @@
+{
+    "manufacturer": "SmithRune",
+    "keyboard_name": "Iron165R2-H",
+    "maintainer": "Gondolindrim",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": false,
+        "extrakey": true,
+        "lto": true,
```

1. `encoder` and `audio` are disabled by default
2. LTO moved to `build`
```suggestion
    "build": {
        "lto": true
    },
    "features": {
        "backlight": false,
        "bootmagic": true,
        "command": false,
        "console": false,
        "extrakey": true,
```